CMakePresets.json

```diff
@@ -0,0 +1,43 @@
+{
+  "version": 6,
+  "include": ["./ConanPresets.json"],
+  "configurePresets": [
+    {
+      "name": "debug-conan",
+      "displayName": "Debug config (Conan)",
+      "description": "Debug config with tests (Conan dependencies used)",
+      "inherits": "conan-debug",
```

Comment:
conan-debug is not present here, same with conan-release

Reply:
`conan install .` creates ConanPresets.json that contains conan-debug and conan-release